docs/linters.md

```diff
@@ -308,6 +309,12 @@ The `nobools` linter checks that fields in the API types do not contain a `bool`
 Booleans are limited and do not evolve well over time.
 It is recommended instead to create a string alias with meaningful values, as an enum.
 
+## NoDurations
+
+The `nodurations` linter checks that fields in the API types do not contain a `Duration` type ether from the `time` package or the `k8s.io/apimachinery/pkg/apis/meta/v1` package.
+
+It's recommended not to use Duration as a type inorder to avoid implementation of go duration parsing.
```

Comment:
What about:
```suggestion
It is recommended to avoid the use of Duration types. Their use ties the API to Go's notion of duration parsing, which may be hard to implement in other languages.

Instead, use an integer based field with a unit in the name, e.g. `FooSeconds`.
```